### PR TITLE
css-fonts - Kristian Krog

### PR DESCRIPTION
**Pull request by krog-kristian**
css-fonts-attempt-2

### CODE REVIEW

File: css-fonts/styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'Fira Sans', sans-serif;
```
Comment (shawnkost) [inline css-fonts/styles.css:2]: You should be using 2 separate fonts. One for the headings, and one for the rest of the content

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Kristian! One change requested below. You should be using 2 separate fonts

**Comment by krog-kristian** [comment]
version2-css-fonts

File: css-fonts/styles.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+body {
+  line-height: 1.6em;
+}
+
+body p {
```
Comment (shawnkost) [inline css-fonts/styles.css:5]: In this instance, you don't actually need to target the `p` here, you can just select the `body`.

File: css-fonts/styles.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  font-family: 'Fira Sans', sans-serif;
+}
+
+body
```
Comment (shawnkost) [inline css-fonts/styles.css:9]: Kinda similar to above, you don't need to target the `body` here, you can just target the `h1` & `h2`. This is related to something called CSS Cascade which we will touch on in a later exercise.

**Review by shawnkost — APPROVED** [review]
This is approved, but I left some notes below to optimize the code. Please take a look and let me know if you have any questions!

**Pull request merged**